app/vlinsert/loki/loki_json.go

```diff
@@ -0,0 +1,133 @@
+package loki
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/valyala/fastjson"
```

Comment:
Could you fix imports please?

Reply:
Do we have some kind of configuration for imports to use?
Seems like I can't figure out exactly what sorting policy should be there :)